configuration/src/test/java/io/camunda/configuration/SecondaryStorageTest.java

```diff
@@ -37,28 +57,61 @@ public class SecondaryStorageTest {
   class WithOnlyUnifiedConfigSet {
     final OperateProperties operateProperties;
     final TasklistProperties tasklistProperties;
+    final BrokerBasedProperties brokerBasedProperties;
 
     WithOnlyUnifiedConfigSet(
         @Autowired final OperateProperties operateProperties,
-        @Autowired final TasklistProperties tasklistProperties) {
+        @Autowired final TasklistProperties tasklistProperties,
+        @Autowired final BrokerBasedProperties brokerBasedProperties) {
       this.operateProperties = operateProperties;
       this.tasklistProperties = tasklistProperties;
+      this.brokerBasedProperties = brokerBasedProperties;
     }
 
-    // @Test
+    @Test
+    @SuppressWarnings("unchecked")
     void testCamundaDataSecondaryStorageType() {
       final DatabaseType expectedOperateDatabaseType = DatabaseType.Elasticsearch;
       assertThat(operateProperties.getDatabase()).isEqualTo(expectedOperateDatabaseType);
 
       final String expectedTasklistDatabaseType = "elasticsearch";
       assertThat(tasklistProperties.getDatabase()).isEqualTo(expectedTasklistDatabaseType);
+
+      /* camundaExporter */
+
+      final ExporterCfg camundaExporter = getCamundaExporter(brokerBasedProperties);
+      assertThat(camundaExporter).isNotNull();
+
+      final Map<String, Object> args = camundaExporter.getArgs();
+      assertThat(args).isNotNull();
+
+      final Map<String, Object> connect = (Map<String, Object>) args.get("connect");
+      assertThat(connect).isNotNull();
+
+      final String type = (String) connect.get("type");
+      assertThat(type).isEqualTo("elasticsearch");
```

Comment:
:x: Verify it via the ExporterConfiguration class. This will ensure there was no typo in the breadcrump used in the override class.

```
final ExporterCfg camundaExporter = getCamundaExporter(brokerBasedProperties);
      assertThat(camundaExporter).isNotNull();

      final Map<String, Object> args = camundaExporter.getArgs();
      assertThat(args).isNotNull();

      final var configuration =
          new ExporterConfiguration("camundaExporter", args)
              .instantiate(io.camunda.exporter.config.ExporterConfiguration.class);

      assertThat(configuration.getConnect().getType()).isEqualTo("elasticsearch");
```

Reply:
:x: To make the tests more readable, can you split the test cases as follows?

1. shouldSetOperateProperties
    * verify operate type and url is set
2. shouldSetTasklistProperties
    * verify tasklist type and url is set
 3. shouldSetCamundaExporter
     * verify type and url
     
  And extend individual test cases when new properties are added.